lib/OrdinaryDiffEqSSPRK/src/alg_utils.jl

```diff
@@ -30,8 +30,21 @@ alg_order(alg::SSPRKMSVS43) = 3
 alg_order(alg::SSPRK932) = 3
 alg_order(alg::SSPRK54) = 4
 alg_order(alg::SSPRK104) = 4
-alg_order(alg::SHLDDRK_2N) = 4
-alg_order(alg::SHLDDRK52) = 2
+
+"""
+    ssp_coefficient(alg)
+
+Return the SSP coefficient of the ODE algorithm `alg`. If one time step of size
+`dt` with `alg` can be written as a convex combination of explicit Euler steps
+with step sizes `cᵢ * dt`, the SSP coefficient is the minimal value of `1/cᵢ`.
+
+# Examples
+
+```julia-repl
+julia> ssp_coefficient(SSPRK104())
+6
+```
+"""
```

Comment:
This needs to go onto the basic dispatch in order to work. It should be on OrdinaryDiffEqCore's

Reply:
Should I shift this docstring to OrdinaryDiffEqCore?